### ISSUE TITLE
Cannot find visdcc as a registered library

### ISSUE THREAD

**Issue by FilomenoSanchez** [rigdenlab/conplot#115]
**Describe the bug**
Dash is unable to import `visdcc` and workers fail to load Plot page. This is only observed the first time the Plot page is requested after a worker fresh start or re-start.

**To Reproduce**
1. Go to conplot.org/plot
2. Page can't load first time, need to wait a few seconds

**Expected behavior**
Dash should be able to find module `visdcc` and Plot page should load without problems.

**Additional context**

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.6/site-packages/dash/dash.py", line 636, in serve_component_suites
    _validate.validate_js_path(self.registered_paths, package_name, path_in_pkg)
  File "/usr/local/lib/python3.6/site-packages/dash/_validate.py", line 292, in validate_js_path
    package_name, list(registered_paths.keys())
dash.exceptions.DependencyException: Error loading dependency. "visdcc" is not a registered library.
Registered libraries are:
['dash_renderer', 'dash_html_components', 'dash_core_components', 'dash_bootstrap_components']
```

**Comment by FilomenoSanchez** [comment]
Probably related to [this](https://community.plotly.com/t/share-visdcc-a-dash-core-components-for-vis-js-network/5417/4) and [this](https://community.plotly.com/t/custom-component-with-multi-page-apps/7145). Need to refactor `import visdcc`  to the initial app.serve_layout

**Comment by FilomenoSanchez** [comment]
Closed by #117
